.prettierrc.yml

```diff
@@ -3,3 +3,4 @@ singleQuote: true
 printWidth: 120
 tabWidth: 4
 bracketSpacing: false
+arrowParens: avoid
```

Comment:
See https://prettier.io/docs/en/options.html#arrow-function-parentheses: 
"First available in v1.9.0, default value changed from avoid to always in v2.0.0"
Because we updated the prettier, I needed to change the option as well.